mail_parent_recipient/tests/test_mail_parent_recipient.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.addons.mail.tests.common import TestMail
+
+
+class TestMailTemplate(TestMail):
+
+    def setUp(self):
```

Comment:
the base test case is a SavepointCase -> move this code to setUpClass to avoid creating these records for every test.